### ISSUE TITLE
Lack of Connectivity between Type A Machines Series 1 (Plywood)

### ISSUE THREAD

**Issue by gchingon** [Jaknil/TAMS12013#1]
I have a Type A Machines Series 1, serial number 00000247. PSU lights up when connected, USB B is plugged into Arduino and the other end is plugged into my Mac. Nothing is detected is in `System Report > Hardware > USB` Perhaps the Arduino is dead/fried? 

1. Is there anything I can test/ohm out before removing/replacing the Arduino (with the low cost of an Arduino should I not bother and just purchase a new one? if so which model? 
2. Is there a firmware/OS I would need to install on a new Arduino?
3. Should I look for/attempt something else before testing/replacing Arduino? (next step is to verify that USB cable is good)

**Comment by Jaknil** [comment]
Yes, you could look into if you have the required drivers to connect to the arduino board. It might not be an official arduino Mega and therfore it might need different USB drivers. I have had to install different USB drivers for different boards on windows machines but I doubt they will work on Mac. (No expierience with mac.) Downloading and installing the software instead of using the web-interface or just un-zipping the official software to your computer helps a lot on PC since that gives you a bunch of typical drivers for various boards, get it on https://www.arduino.cc/ 

Yes I have the stock frimware for this type of printer on this repo. [Download and install this if you need it. ](https://github.com/Jaknil/TAMS12013/blob/master/STOCK_TAM_Series_1_Marlin_Firmware_RevB.zip)
It's a speical configuration of the very common open source RAMPS 1.4 3D printer controllers so you can google for generic installations guides for RAMPS firmware.

Things to test: Does the power on LED come on the arduino board when you connect the USB to your computer? It can not be disabled so if it is off, then something is wrong. It is close to the reset button and marked ON
![image](https://user-images.githubusercontent.com/3476653/103747277-aeef3100-5002-11eb-9352-76cbb838a66e.png)
You could also measure the voltage between a 5V pin and ground but it comes to the same thing as just looking at the power LED. 

I also have [the manual for this printer uploaded here](https://github.com/Jaknil/TAMS12013/blob/master/Series12013StandardUserManualV1_6.pdf)

Good luck and remember to report back here if you find the solution so it may help others.
